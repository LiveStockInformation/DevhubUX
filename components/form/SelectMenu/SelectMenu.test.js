import * as React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'

import { SelectMenu } from '.'

afterEach(cleanup)

describe('SelectMenu Component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(
      <SelectMenu
        label='Test'
        id='Test'
        items={[
          {
            label: 'Item 1',
            value: '1'
          },
          {
            label: 'Item 2',
            value: '2'
          }
        ]}
      />
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
