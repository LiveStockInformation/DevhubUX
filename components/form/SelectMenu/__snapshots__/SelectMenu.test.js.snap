// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectMenu Component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="ls-form-group  "
  >
    <div
      class="ls-select"
    >
      <label
        class="ls-label "
        for="Test"
      >
        Test
      </label>
      <select
        class="ls-select"
        id="Test"
      >
        <option
          value="1"
        >
          Item 1
        </option>
        <option
          value="2"
        >
          Item 2
        </option>
      </select>
    </div>
  </div>
</DocumentFragment>
`;
